src/test/java/Homework20.java

```diff
@@ -0,0 +1,24 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+    @Test
+    public static void createNewPlaylistTest() {
+
+        logIn("andalia.nia@gmail.com", "te$t$tudent");
+        clickSubmit();
+        createNewPlaylist();
+        Assert.assertTrue(newPlaylistDisplayed());
+    }
+    @Test
+    public static void addASongToPlaylist(){
+        logIn("andalia.nia@gmail.com","te$t$tudent");
+        clickSubmit();
+        searchSong("Pluto");
+        clickViewAllButton();
+        selectFirstSong();
+        clickAddToButton();
+        Assert.assertTrue(newPlaylistDisplayed());
+    }
+}
```

Comment:
Test error. You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225227784-b0bb1bfc-f231-47a2-9695-ad6aee27ba4f.png)

HINT

![Test status](https://user-images.githubusercontent.com/104761482/225228195-351e9450-e3d2-486b-a764-17a883ec5644.png)

![Test error_](https://user-images.githubusercontent.com/104761482/225228243-bf5e1e75-3609-4696-9a97-97c1e804747a.png)